irohad/main/impl/consensus_init.cpp

```diff
@@ -45,8 +45,9 @@ namespace iroha {
         return crypto;
       }
 
-      auto YacInit::createTimer() {
-        return std::make_shared<TimerImpl>();
+      auto YacInit::createTimer(std::chrono::milliseconds delay_milliseconds) {
+        return std::make_shared<TimerImpl>(rxcpp::observable<>::timer(
+            delay_milliseconds, rxcpp::observe_on_new_thread()));
```

Comment:
Each timer with emit new thread? So, Is it possible to rework it with a scheduler that handles only one thread?